vulcanai2/models/basenetwork.py

```diff
@@ -1,17 +1,26 @@
 # -*- coding: utf-8 -*-
 """Defines the basenetwork class"""
+# Core imports
 import abc
 from torch.autograd import Variable
 import torch.nn.modules.loss as loss
+
+# Vulcan imports
 from .layers import *
 from .metrics import Metrics
+from vulcanai2.plotters.visualization import display_record
```

Comment:
you should do relative imports here

Reply:
something like `from ..plotters.visualization import display_record` ?

Reply:
yeah